src/tasks/barfTasks.ts

```diff
@@ -264,76 +259,82 @@ const turns: AdventureAction[] = [
           Macro.externalIf(underwater, Macro.item($item`pulled green taffy`)).meatKill()
         ).abort()
       );
-      return get("_sourceTerminalDigitizeMonsterCount") !== start;
+      return;
     },
     spendsTurn: () => !SourceTerminal.getDigitizeMonster()?.attributes.includes("FREE"),
     sobriety: Sobriety.EITHER,
+    completed: () => get("_sourceTerminalDigitizeMonsterCount") !== digitize,
```

Comment:
So I can imagine one failure case here:

Someone runs garbo with relatively few turns left, and it redigitizes after 1 monster. Or wait does that not work as a failure case?

I think that actually is fine, just putting it out there.